css/style.css

```diff
@@ -0,0 +1,486 @@
+  width: 80px;
+  height: 24px;
+  margin-top: 10px;
+}
+
+.col2 {
+  flex-direction: row;
+  flex: 0.7;
+  margin-top: 10px;
+  justify-content: center;
+}
+
+.col2 > form {
+  flex: 1;
+  height: 32px;
+}
+
+.col2 .searchbox {
+  margin-right: 0;
+  padding-right: 0;
+  flex-grow: 0.7;
+  border-style: solid;
+  border-radius: 0;
+  border-right: 0;
+  box-shadow: none !important;
+}
+
+.col2 .searchBtn {
+  margin-left: 0 !important;
+  border-style: solid;
+  border-radius: 0;
+  cursor: pointer;
+}
+
+.comments-icon > svg {
+  width: 24px;
+  height: 24px;
+
+  /* margin-right: 10px; */
+  fill: #5c5a5a;
+}
+
+.save-icon > a > i > svg {
+  width: 20px;
+  height: 20px;
+  fill: #666464;
+  vertical-align: middle;
+  display: inline-block;
+}
+
+.col3 .icon-wrapper svg {
+  fill: #606060;
+}
+
+.col2 .searchBtn svg {
+  width: 25px;
+  height: 25px;
+  fill: rgb(165, 161, 161);
+}
+
+.col2 .searchBtn svg:hover {
+  fill: rgb(122, 116, 116);
+}
+
+.col3 {
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.col3 .icon-wrapper .avatar {
+  border-radius: 60px;
+}
+
+.icon-wrapper {
+  width: 24px;
+  height: 24px;
+  padding: 0 15px;
+}
+
+.page-wrapper {
+  flex-direction: row;
+  background: #f8f8fa;
+  padding: 20px;
+}
+
+.main-content {
+  width: 70%;
+  margin: 20px 20px;
+}
+
+/* video stats */
+.video-stats {
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.views {
+  margin: 10px;
+}
+
+.likes-dislikes {
+  flex-direction: row;
+}
+
+.likes-dislikes > div > button,
+.likes-dislikes > div > a,
+.likes-dislikes > .likes {
+  margin: 5px;
+  background: transparent;
+  border: none;
+}
+
+.likes-dislikes > .save-icon > button {
+  margin: 5px;
+  background: transparent;
+  border: none;
+}
+
+.uploader-info {
+  flex-direction: row;
+}
+
+.uploader-icon {
+  margin-top: 5px;
+  width: 7%;
+}
+
+.channel-info {
+  width: 60%;
+}
+
+.channel-info .uploader-name {
+  font-size: 1rem;
+  font-weight: bold;
+  margin-bottom: 1px;
+}
+
+.channel-info .subscriber-info {
+  font-size: 0.9rem;
+  margin-top: 5px;
+}
+
+div.subscriber-button {
+  justify-content: flex-end;
+  width: 33%;
+  align-items: flex-start;
+}
+
+div.subscriber-button > a {
+  text-transform: uppercase;
+  border-style: solid;
+  border-radius: 5%;
+  display: block;
+  width: 120px;
+  text-align: center;
+  padding: 10px;
+}
+
+/* video info */
+.video-info {
+  flex-direction: row;
+  padding-left: 7%;
+}
+
+.video-content {
+  width: 60%;
+}
+
+.video-content > .description {
+  font-size: 0.9rem;
+}
+
+
+/* video icon styles */
+.uploader-icon > img {
+  border-radius: 50%;
+}
+
+/* channel info styles */
+
+/* comments stats */
+.comments-stats {
+  flex-direction: row;
+}
+
+.comments-num {
+  width: 20%;
+}
+
+.comments-num > p {
+  font-size: 1rem;
+}
+
+.comments-sorting {
+  flex-direction: row;
+  width: 85%;
+  align-items: center;
+}
+
+.comments-icon {
+  width: 6%;
+
+  /* padding-top: 14px; */
```

Comment:
[OPTIONAL] It's optional, since it wasn't pointed earlier, but I strongly suggest you guys to implement it.
- [ ] Comments about the code are allowed, commented code not. Please take care of it boys 💪